arch/arm/src/lc823450/lc823450_timer.c

```diff
@@ -148,6 +148,8 @@ static void hrt_usleep_add(struct hrt_s *phrt);
  * Private Data
  ****************************************************************************/
 
+static spinlock_t g_rtc_lock = SP_UNLOCKED;
```

Comment:
```suggestion
static spinlock_t g_hrt_timer_queue_lock = SP_UNLOCKED;
```
and move to line 141

Reply:
done